src/components/SampleDropdown.tsx

```diff
@@ -3,17 +3,15 @@ import { DownOutlined } from "@ant-design/icons";
 import { useCallback, useMemo, useState } from "react";
 import useAppStore from "../store/store";
 import { shallow } from "zustand/shallow";
-import { useStoreWithEqualityFn } from "zustand/traditional";
 
 function SampleDropdown({
   setLoading,
 }: {
   setLoading: React.Dispatch<React.SetStateAction<boolean>>;
 }): JSX.Element {
-  const { samples, loadSample } = useStoreWithEqualityFn(
```

Comment:
`useStoreWithEqualityFn` was recently added in #128 by @nitro56565. What is the reasoning for removing it?